services/galley/test/integration/API.hs

```diff
@@ -417,13 +419,15 @@ postConvWithRemoteUsersOk rbs = do
 
     liftIO $
       -- as many federated requests as remote backends, i.e., one per remote backend
-      Set.size (Set.fromList $ frTargetDomain <$> federatedRequests) @?= Set.size rbs
+      Set.size (Set.fromList $ frTargetDomain <$> federatedRequests') @?= Set.size rbs
+    for_ federatedRequests' print
```

Comment:
can you remove this noise and either not print anything, or use the test framework's output channels?

Reply:
Oh, that was unintentional.